docs/reference/run.md

```diff
@@ -345,7 +345,7 @@ Supported networks :
       <td class="no-wrap"><strong>container</strong>:&lt;name|id&gt;</td>
       <td>
         Use the network stack of another container, specified via
-        its *name* or *id*.
```

Comment:
Ah, yes, that's due to the switch to Jekyll 😞 , which doesn't support Markdown in HTML tables; I think this is being addressed in https://github.com/docker/docker/pull/26791